app/javascript/components/subject/CurrentStatus.js

```diff
@@ -50,19 +51,40 @@ class CurrentStatus extends React.Component {
     }
   }
 
+  generateReportEligibility(eligibility) {
+    return (
+      <React.Fragment>
+        {eligibility.eligible && (
+          <span>
+            . This {this.props.isolation ? 'case' : 'monitoree'} is currently eligible to receive a notification today&nbsp;
+            <EligibilityTooltip report_eligibility={this.props.report_eligibility} id={`eltt`} inline={true} />.
+          </span>
+        )}
+        {!eligibility.eligible && (
+          <span>
+            . This {this.props.isolation ? 'case' : 'monitoree'} is not currently eligible to receive a notification today&nbsp;
+            <EligibilityTooltip report_eligibility={this.props.report_eligibility} id={`eltt`} inline={true} />.
```

Comment:
You can avoid some code duplication here by having an `eligibleText` variable (or whatever you name it) you set above based on `eligibility.eligible`. Then you only need one `span`.

Reply:
Agreed, done.